tests/unit/test_file_operation.py

```diff
@@ -1,10 +1,12 @@
 #
 # Copyright (c) 2012-2023 Snowflake Computing Inc. All rights reserved.
 #
+import pytest
 
 from snowflake.snowpark import GetResult, PutResult
 
 
+@pytest.mark.localtest
```

Comment:
This is not necessary, all unit tests are run against Local Testing.

Reply:
removed